### PR TITLE
Add Red Hat Satellite 6.6 and clean-up code

### PR DESCRIPTION
**Pull request by stdevel**
This PR fixes the following:
- adds missing Red Hat Satellite 6.6 GA content (*see also issue #29*) and sets it as default
- cleans-up the code, fixes some ``ansible-lint`` findings
- disables the EPEL repository before installing Satellite to avoid package violations (*see also issue #20*)
- disables auto-provisioning of RHEL 7.6 content as not every user wants to have it (*it's just commented out so that users can see how to implement this, maybe we should add this to the documentation?*)

### CODE REVIEW

**Comment by stdevel** [comment]
Successfully tested the role with Satellite 6.5 and 6.6. 🤙

**Comment by stdevel** [comment]
Hello,
are there some news? As work on Satellite 6.7 already began, it would be great having this PR merged.

Best wishes,
Christian.

**Comment by itewk** [comment]
@stdevel appologies, i didn't see this. instead of commenting lines out can you just delete them please.

File: defaults/main.yml [hunk 1]
```diff
@@ -34,14 +34,14 @@ satellite_deployment_answers:
 satellite_deployment_manifest_path: "satellite_manifest.zip"
 satellite_deployment_manifest_dest_path: "/root/satellite_manifest.zip"
 satellite_deployment_repositories:
-  - { product_name: 'Red Hat Enterprise Linux Server', name_repo: 'Red Hat Enterprise Linux 7 Server (Kickstart)', rel: '7.6', architecture: 'x86_64', state: enable}
-  - { product_name: 'Red Hat Enterprise Linux Server', name_repo: 'Red Hat Enterprise Linux 7 Server (Kickstart)', rel: '7Server', architecture: 'x86_64', state: enable}
-  - { product_name: 'Red Hat Enterprise Linux Server', name_repo: 'Red Hat Enterprise Linux 7 Server (RPMs)', rel: '7Server', architecture: 'x86_64', state: enable}
-  - { product_name: 'Red Hat Enterprise Linux Server', name_repo: 'Red Hat Enterprise Linux 7 Server - RH Common (RPMs)', rel: '7Server', architecture: 'x86_64', state: enable}
+  #- { product_name: 'Red Hat Enterprise Linux Server', name_repo: 'Red Hat Enterprise Linux 7 Server (Kickstart)', rel: '7.6', architecture: 'x86_64', state: enable}
```
Comment (itewk) [inline defaults/main.yml:37]: if this is no longer needed please delete rather then comment out.

**Comment by stdevel** [comment]
Hey @itewk,
unfortunately I accidently removed my source repository. 🤕
So one possibility would be if you remove the commented out lines. I also could fork the repository for a second time and re-do my changes.

Which way would you prefer?

**Review by itewk — APPROVED** [review]

**Comment by itewk** [comment]
@stdevel merged and did a second commit to remove commented out code.

**Pull request merged**